src/lib/campusSafetyRouting.ts

```
@@ -0,0 +1,468 @@
+ */
+
+import {
+  SafetyHeatmapPoint,
+  SafetyReport,
+  GeoLocationPoint,
+  CampusInfrastructureNode,
+  SafeRouteComparison,
+  SafeWalkRouteResult,
+  SafeRouteSegment,
+  SafetyZoneRisk,
+} from '../types/campusSafety';
```

ESLint

[error] Replace `'../types/campusSafety'` with `"../types/campusSafety"`

(prettier/prettier)